src/components/CheckBox.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+const CheckBox = props => {
```

Comment:
What's the reasoning behind creating this component insteads  of  using https://baltimorecounty.github.io/dotgov-components/#checkbox ?

Reply:
converted to component and also name change to stay consistent with what its doing